pontoon/admin/urls.py

```diff
@@ -0,0 +1,16 @@
+from django.conf.urls import include, patterns, url
+
+from pontoon.admin import views
+from pontoon.admin.site import admin_site
+
+
+urlpatterns = patterns(
+    '',
```

Comment:
Wouldn't that affect the admin URLs too, though?

I actually wonder if I shouldn't include the custom views in the `admin_site` object as well, so that they're consistent with the `admin:` namespace and naming style. Hmmm....